VideoWeb/VideoWeb.UnitTests/EventHandlers/PauseEventHandlerTests.cs

```diff
@@ -9,6 +9,33 @@
 
 namespace VideoWeb.UnitTests.EventHandlers
 {
+    public class StartEventHandlerTests : EventHandlerTestBase
+    {
+        private StartEventHandler _eventHandler;
+
+        [Test]
+        public async Task Should_send_messages_to_participants_on_pause()
```

Comment:
Shouldn't this be on a resume rather than pause?

Reply:
fixed the test names and moved into their own files.